proposals/evaluation/uc-3-inheritance.md

```diff
@@ -64,8 +63,18 @@ The access control `</weekly-status/.acp#ac1>` applies to all resources containe
   acp:applyMembers </acp/research#p1> . # applies the policy to all resources contained by </weekly-status/>
 ```
 
-Todo:  what is the ACR for 
-`</weekly-status/2021-04-28/report.md>` and what information is in it?
+An unauthenticated client making a GET on `</weekly-status/>` container will receive a `Link: <.acp>; rel="access-control"` header in the `401` response that points to the above `<.acp>`. 
```

Comment:
For now, given the state of the web-access-control specification and since we're aligning, the rel is `acl`.
```suggestion
An unauthenticated client making a GET on `</weekly-status/>` container will receive a `Link: <.acp>; rel="acl"` header in the `401` response that points to the above `<.acp>`. 
```